Extractor/UAClient.cs

```diff
@@ -1463,9 +1463,24 @@ public static double ConvertToDouble(object datavalue)
             if (!nsPrefixMap.TryGetValue(nodeId.NamespaceIndex, out var prefix))
             {
                 var namespaceUri = id.NamespaceUri ?? NamespaceTable!.GetString(nodeId.NamespaceIndex);
-                string newPrefix = namespaceUri is not null
-                    && Config.Extraction.NamespaceMap.TryGetValue(namespaceUri, out string prefixNode)
-                    ? prefixNode : (namespaceUri + ":");
+                string newPrefix;
+                if (namespaceUri is not null)
+                {
+                    if (Config.Extraction.NamespaceMap.TryGetValue(namespaceUri, out string prefixNode))
+                    {
+                        newPrefix = prefixNode;
+                    }
+                    else
+                    {
+                        newPrefix = namespaceUri + ":";
+                    }
+                }
+                else
+                {
+                    log.LogWarning("NodeID received with null NamespaceUri, and index not in NamespaceTable. This should not be possible! {Id}, {Index}",
```

Comment:
I think this should be more descriptive .
```suggestion
                    log.LogWarning("NodeID received with null NamespaceUri, and index not in NamespaceTable. Unprocessable namepace! {Id}, {Index}",
```

Reply:
That doesn't seem more descriptive as much as the same with more technical language, but I can agree that the wording isn't very clear. I wanted to avoid pointing fingers too hard. Perhaps `This is likely a bug in the server`?